src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/CodeActions/Razor/ExtractToCodeBehindCodeActionResolverTest.cs

```diff
@@ -126,8 +126,8 @@ public async Task Handle_ExtractCodeBlock()
             Uri = documentPath,
             RemoveStart = contents.IndexOf("@code", StringComparison.Ordinal),
             ExtractStart = contents.IndexOf("{", StringComparison.Ordinal),
-            ExtractEnd = contents.IndexOf("}", StringComparison.Ordinal),
-            RemoveEnd = contents.IndexOf("}", StringComparison.Ordinal),
+            ExtractEnd = contents.IndexOf("}", StringComparison.Ordinal) + 1,
+            RemoveEnd = contents.IndexOf("}", StringComparison.Ordinal) + 1,
```

Comment:
Why do these all require a `+ 1` now?

Reply:
In `contents` we had a "}" that was being cut off without the `+ 1`

Reply:
That makes sense -- thanks! It's probably worth it to add a helper and use it in each location to make it clear what's happening.